docs/index.md

```diff
@@ -1,5 +1,5 @@
 # Welcome to OpenDI 
-We're glad you're here!  The purpose of this website is to publish OpenDI standards and assets for you to use and contribute to.  
+We're glad you're here!  The purpose of this website is to publish OpenDI standards and resources for you to use and contribute to.  
```

Comment:
Suggest you replace 'standards and resources' with 'standard, materials, and resources'.

Reply:
Done in a0cc2c7